src/functions-and-arrays.js

```diff
@@ -1,41 +1,89 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(n1, n2) {
+  if (n1 > n2) return n1;
```

Comment:
Nice one Nuno! 🏆

Another way to solve this one would be to use the Math.max() method. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)

Or maybe even attempt solving it with the all mighty [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)! 😏